Example/TABObserverSet/ViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  TABObserverSet
+//
+//  Created by KaneCheshire on 05/02/2017.
+//  Copyright (c) 2017 KaneCheshire. All rights reserved.
+//
+
```

Comment:
Not yet, but was thinking about putting a few examples in there as a separate PR. Didn't want to clog this one up too much as it's messy anyway